### PR TITLE
added shortcut filtering

### PR DESCRIPTION
**Pull request by folbo**
Textbox for input placed on the bottom of preferences dialog.

### CODE REVIEW

File: mscore/prefsdialog.ui [hunk 1]
```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>712</width>
```
Comment (lasconic) [inline mscore/prefsdialog.ui]: Apparently Qt Creator did some change behind your back. Could you reset the size and the default tab?
Also remove the & in the strings?

**Comment by lasconic** [comment]
Thank you for the PR !
I guess the "right" way would be to replace the QTreeWidget by a QTreeView and use QSortFilterProxyModel... but if it works, let use this.

File: mscore/prefsdialog.ui (hunk 1 above)
Comment (folbo) [inline mscore/prefsdialog.ui]: sure, didn't notice

**Comment by folbo** [comment]
Changed back dialog design and named commit properly, should be fine now

**Comment by lasconic** [comment]
looks better. If you could squash both commits. That would be perfect! http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by folbo** [comment]
Thank you, always something new :) Squashed.

**Pull request merged**